specification/loadtestservice/LoadTestService/routes.tsp

```diff
@@ -0,0 +1,488 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/openapi";
+import "./models.tsp";
+import "@azure-tools/typespec-autorest";
+
+using Azure.Core;
+using Azure.Core.Traits;
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+using Autorest;
+using OpenAPI;
+
+namespace Microsoft.LoadTestService;
+
+@tag("Test")
+interface LoadTestAdministration {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @summary("Create a new test or update an existing test by providing the test Id.")
+  @doc("Create a new test or update an existing test by providing the test Id.")
+  @route("/tests/{testId}")
+  @patch
+  createOrUpdateTest is Azure.Core.Foundations.Operation<
```

Comment:
Why couldn't you use `ResourceOperations.ResourceCreateOrUpdate` here?

Reply:
Because it is changing the request body. We expect a json body in a parameter named body. After using standard operation, it changes to parameter name "resource".

Reply:
If it only changes the parameter name then it is still preferable to use the Azure Core operation template.

Reply:
We have sdk based out of these specs which expects named parameter body in operations. Have raised an issue - https://github.com/Azure/typespec-azure/issues/3742 regarding this.

Reply:
Given the discussion in the issue (https://github.com/Azure/typespec-azure/issues/3742) let's leave this as is for now and update later on if the template supports updating the parameter name